src/Services/Tags/MixinTag.php

```diff
@@ -6,6 +6,14 @@
 
 class MixinTag extends AbstractDescriptionTag
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function getTagName(): string
+    {
+        return 'mixin';
+    }
+
```

Comment:
I'm not sure why the mixin class would need a tag name

Reply:
for short name in sorting list, without this, you need use 'romanzipp\modeldoc\services\tags\mixin'

Reply:
Alright, got it!